src/Castle.Windsor.Extensions.DependencyInjection/Scope/ExtensionContainerScope.cs

```diff
@@ -20,36 +20,40 @@ namespace Castle.Windsor.Extensions.DependencyInjection.Scope
 	using Castle.Core;
 	using Castle.MicroKernel;
 	using Castle.MicroKernel.Lifestyle.Scoped;
-	
+
 	internal class ExtensionContainerScope : ILifetimeScope, IDisposable
 	{
 		public static ExtensionContainerScope Current => current.Value;
 		public static string TransientMarker = "Transient";
 		protected static readonly AsyncLocal<ExtensionContainerScope> current = new AsyncLocal<ExtensionContainerScope>();
 		private readonly ExtensionContainerScope parent;
+		private readonly ExtensionContainerRootScope rootScope;
```

Comment:
Field `rootScope` is unused.

Reply:
it is used there
![image](https://user-images.githubusercontent.com/2411563/101457781-3da25c80-3936-11eb-8f44-1389e6cb1667.png)

Reply:
cleaned up now